torchrl/envs/libs/vmas.py

```diff
@@ -94,6 +94,17 @@ def _vmas_to_torchrl_spec_transform(
                 device=device,
             )
         )
+    elif isinstance(spec, gym_spaces.Dict):
+        spec_out = {}
+        for k in spec.keys():
+            key = k
```

Comment:
what does this do?

Reply:
good question. I have no idea why that was in my code. I hope it was not there for some dark magic that i forgot about